Client/Assets/CSS/forms.scss

```diff
@@ -0,0 +1,4 @@
+input[type=checkbox],
+input[type=radio]::before {
+border-color: $watermelon !important;
```

Comment:
Indentation? :)

Reply:
Is this file actually included in the build?

Reply:
good catch @LukasReschke 😸 , it's been committed by accident 🙈